doc/man3/CMS_get0_type.pod

```diff
@@ -26,10 +27,14 @@ flag and B<before> the structure is finalised, otherwise the results are
 undefined.
 
 ASN1_OBJECT *CMS_get0_eContentType() returns a pointer to the embedded
-content type.
+content type.  It differs from CMS_get0_type in that it points to the actual
+eContentType element contained in the ASN1 structure that was constructed or received.
+It should return a single encoded OID.
 
 CMS_get0_content() returns a pointer to the B<ASN1_OCTET_STRING> pointer
-containing the embedded content.
+containing the embedded content itself.
+It's use is undefined on outgoing objects created with B<CMS_PARTIAL>
+until L<CMS_final> is called.
```

Comment:
This statement might be incorrect.  I'm unclear what the API contract should be here.

Reply:
I'm not sure that statement is correct. AFAICT (from code inspection only), I think you will get back a valid \*\* pointer, but then thing it points to will be NULL, indicating that the embedded content has not been created yet.